MAESTRO/scATAC_utility.py

```diff
@@ -123,4 +123,4 @@ def universal_open ( filename, mode ):
     if is_gzip( filename ):
         return gzip.open( filename, mode )
     else:
-        return open( filename, mode )
+        return open( filename, mode )
```

Comment:
Missing newline at the end of file

Reply:
I will add a newline at the end of the file. Thanks!